source/layout/theme.liquid

```diff
@@ -10,6 +10,11 @@
 
   <meta name="viewport" content="width=device-width, initial-scale=1">
 
+  <script>
+    // remove no-js and add 'js' to the HTML
```

Comment:
good point. no use for it in the most basic boilerplate.  but i appreciate you keeping it in, as it's something i use on all my projects and can't possibly hurt to keep it there as a reminder to other devs to progressively enhance their work. :)